src/view.c

```diff
@@ -1530,29 +1567,21 @@ view_get_adjacent_output(struct view *view, enum view_edge edge)
 		return NULL;
 	}
 
+	struct wlr_box box = output_usable_area_in_layout_coords(view->output);
```

Comment:
Could just be `output`, it is defined above.

Reply:
Good spot. Done.